src/util/value.ts

```diff
@@ -24,6 +24,8 @@ export type Camera = {
   fov: number;
 };
 
+export type SceneMode = "scene3d" | "scene2d" | "columbus";
```

Comment:
Either this is in the wrong spot, or you need to add this to the valueTypeMapper below as it asks. 
(Pretty sure the types here are for the typescript versions of GraphQL types and we use these + the mapper in the `valueFromGQL` function to convert. So if thats not how this type is being used, move it somewhere more appropriate.)

Reply:
Sorry didn't notice this is for convert values. Moved into the `ref.ts` now.